modules/logging-common-terms.adoc

```diff
@@ -0,0 +1,83 @@
+
+Cluster Logging Operator (CLO)::
+The Cluster Logging Operator provides a set of APIs to control the collection and forwarding of application, infrastructure, and audit logs.
+
+Custom Resource (CR)::
+A CR is an extension of the Kubernetes API. To configure {product-title} Logging and log forwarding, you can customize the `ClusterLogging` and the `ClusterLogForwarder` custom resources.
+
+event router::
+The event router is a pod that watches {product-title} events. It collects logs by using {product-title} Logging.
+
+Fluentd::
+Fluentd is a log collector that resides on each {product-title} node. It gathers application, infrastructure, and audit logs and forwards them to different outputs.
+
+garbage collection::
+Garbage collection is the process of cleaning up cluster resources, such as terminated containers and images that are not referenced by any running pods.
+
+Elasticsearch::
+Elasticsearch is a distributed search and analytics engine. {product-title} uses ELasticsearch as a default log store for {product-title} Logging.
+
+Elasticsearch Operator::
+Elasticsearch operator is used to run Elasticsearch cluster on top of {product-title}. The Elasticsearch Operator provides self-service for the Elasticsearch cluster operations and is used by {product-title} Logging.
+
+indexing::
+Indexing is a data structure technique which is used to quickly locate and access data. Indexing optimizes the performance by minimizing the amount of disk access required when a query is processed.
+
+JSON logging::
+{product-title} Logging Log Forwarding API enables you to parse JSON logs into a structured object and forward them to either {product-title} Logging-managed Elasticsearch or any other third-party system supported by the Log Forwarding API.
+
+Kibana::
+Kibana is a browser-based console interface to query, discover, and visualize your Elasticsearch data through histograms, line graphs, and pie charts.
+
+Kubernetes API server::
+Kubernetes apiserver validates and configures data for the API objects.
+
+Labels::
+Labels are key-value pairs that you can use to organize and select subsets of objects such as a pod.
+
+Logging::
+With {product-title} Logging you can aggregate application, infrastructure, and audit logs throughout your cluster. You can also store them to a default log store, forward them to third party systems, and query and visualize the stored logs in the default log store.
+
+logging collector::
+A logging collector collects logs from the cluster, formats them, and forwards them to the log store or third party systems.
+
+log store::
+A log store is used to store aggregated logs. You can use the default Elasticsearch log store or forward logs to external log stores. The default log store is optimized and tested for short-term storage.
+
+log visualizer::
+Log visualizer is the user interface (UI) component you can use to view logs, graphs, charts, and so forth. The current implementation is Kibana.
```

Comment:
See the ISG guidance on _[and so on](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#and-so-on)_. WDYT of the following:

```suggestion
Log visualizer is the user interface (UI) component you can use to view information such as logs, graphs, charts, and other metrics. The current implementation is Kibana.
```

Reply:
acked